scripts/localnet-2.sh

```diff
@@ -53,20 +53,16 @@ main() {
         --path "${_tmp}/unsigned.toml" \
         --output "${_tmp}/signed.toml" \
         --alias validator-1
-    cat "${_tmp}/signed.toml" >> "${_tmp}/localnet/transactions.toml"
-
-    # attribute balance to new validator
-    sed \
-        -i '' \
-        's/\(\[token\.BTC\]\)/tnam1qyq850fy0tdk8wkp40hhwu8a9wp2wn8stq3ldqrg = "200000"\ntnam1qr8l7l6rywucdarxg9q0zpggfe0jxddk6u09e8ez = "1000000"\n\1/' \
-        "${_tmp}/localnet/balances.toml"
+    cat "${_tmp}/signed.toml" >>"${_tmp}/localnet/transactions.toml"
 
     # generate localnet
     ./scripts/gen_localnet.py \
         --full-nodes '{"fullnode-0":12340}' \
         --templates "${_tmp}/localnet" \
         --validator-aliases '{"validator-0":"tnam1q9vhfdur7gadtwx4r223agpal0fvlqhywylf2mzx","validator-1":"tnam1qyq850fy0tdk8wkp40hhwu8a9wp2wn8stq3ldqrg"}' \
         --pre-genesis-path "${_tmp}/localnet/src/pre-genesis" \
+        --edit-temp '{"balances.toml":{"token":{"NAM":"insert_dict(it,tnam1qyq850fy0tdk8wkp40hhwu8a9wp2wn8stq3ldqrg=\"200000\",tnam1qr8l7l6rywucdarxg9q0zpggfe0jxddk6u09e8ez=\"1000000\")"}}}' \
```

Comment:
```suggestion
        --edit-templates '{"balances.toml":{"token":{"NAM":"insert_dict(it,tnam1qyq850fy0tdk8wkp40hhwu8a9wp2wn8stq3ldqrg=\"200000\",tnam1qr8l7l6rywucdarxg9q0zpggfe0jxddk6u09e8ez=\"1000000\")"}}}' \
```

Reply:
the script still works when you abbreviate the flags

Reply:
funky